code/modules/projectiles/projectile/energy/stun.dm

```diff
@@ -15,6 +15,12 @@
 	muzzle_type = /obj/effect/projectile/muzzle/stun
 	impact_type = /obj/effect/projectile/impact/stun
 
+/obj/item/projectile/beam/disabler/pass_glass
+		name = "beam-disabler"
+		damage = 50
+		damage_type = STAMINA
```

Comment:
this should probably go wherever the base disabler baem type is rather than in the energy electrode file

Reply:
oh yea, i didnt notice